### ISSUE TITLE
profvis tabs are incorrectly themed

### ISSUE THREAD

**Issue by kevinushey** [rstudio/rstudio#3630]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.1038
    OS Version      : macOS 10.14
    R Version       : 3.5.1

### Steps to reproduce the problem

Run e.g.

```R
library(dplyr)
mtcars <- as_data_frame(mtcars)
profvis::profvis(print(mtcars))
```

### Describe the problem in detail

The dark theme seems to be incorrectly or inconsistently applied:

<img width="874" alt="screen shot 2018-10-08 at 8 51 48 pm" src="https://user-images.githubusercontent.com/1976582/46645969-5cc9b780-cb3c-11e8-9308-13a5e486f624.png">

### Describe the behavior you expected

View should concord with v1.1.

<img width="432" alt="screen shot 2018-10-08 at 8 54 15 pm" src="https://user-images.githubusercontent.com/1976582/46645973-618e6b80-cb3c-11e8-8a40-715475c9f189.png">

**Comment by kevinushey** [comment]
IIUC this was fixed by https://github.com/rstudio/rstudio/commit/8a0cde51fa6cb7e57dfd17474073ed3a412915b2 -- thanks @MariaSemple!

**Comment by dfalty** [comment]
Verified fixed on version 1.2.1090.
